components/overflow-group/overflow-group-mixin.js

```diff
@@ -370,8 +371,7 @@ export const OverflowGroupMixin = superclass => class extends LocalizeCoreElemen
 			await this._getItems();
 
 			this._slotItems.forEach(item => {
-				const observer = new MutationObserver(this._handleItemMutation);
```

Comment:
In this code, we would create a new `MutationObserver` (for every item in the slot, which wasn't necessary), but we would never actually disconnect them on future `slotchange` events.